# ftp://invisible-island.net/temp/xterm-278-p2.patch.gz
# patch by Thomas E. Dickey <dickey@invisible-island.net>
# created  Tue Nov 27 10:17:13 UTC 2012
# ------------------------------------------------------------------------------
# misc.c |    2 +-
# 1 file changed, 1 insertion(+), 1 deletion(-)
# ------------------------------------------------------------------------------
Index: misc.c
--- xterm-278+/misc.c	2012-01-07 01:57:52.000000000 +0000
+++ xterm-278-p2/misc.c	2012-11-27 10:00:55.000000000 +0000
@@ -3770,7 +3770,7 @@
 };
 
 #define MdBool(bool)      ((bool) ? mdMaybeSet : mdMaybeReset)
-#define MdFlag(mode,flag) MdBool(xw->keyboard.flags & MODE_KAM)
+#define MdFlag(mode,flag) MdBool((mode) & (flag))
 
 /*
  * Reply is the same format as the query, with pair of mode/value:
